app/components/AccountSetting/AccountSettingSection.js

```diff
@@ -18,14 +30,18 @@ export default class AccountSettingSection extends Component {
     });
   }
 
+  handleEmailChange(event) {
+    this.setState({ email: event.target.value});
+  }
+
   render() {
-    const { email, children } = this.props;
+    const { location, email, children } = this.props;
```

Comment:
please add location into `propTypes` because this will make `eslint` fail. Also, install `eslint` on your text editor or ide